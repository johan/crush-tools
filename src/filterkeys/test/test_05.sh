test_number=05
description="stdin filter-file"

outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

cat "$test_dir/test-filter.in" |
$bin -p -a 1,2 -b 2,3 -f - "$test_dir/test-3.in" "$test_dir/test-4.in" > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
