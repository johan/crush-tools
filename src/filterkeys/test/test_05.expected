value1^key1^key2^value2
1234^aaa^aaa^98.765
1234^aaa^aab^56.341
3491^ccv^aab^56.341
3491^ddd^erb^56.341
3491^wwz^awz^98.765
3491^fgh^awz^98.765
3491^kwq^erb^98.765
4564^aaa^aab^56.341
5671^ccv^aab^56.341
3491^ddd^erb^56.341
3451^wwz^awz^98.765
2341^fgh^awz^98.765
5431^kwq^erb^98.765
1234^aaa^aaa^98.765
1234^aaa^aab^56.341
3491^ccv^aab^56.341
3491^ddd^erb^56.341
3491^wwz^awz^98.765
3491^kwq^erb^98.765
3491^wwz^awz^98.765
3491^kwq^erb^98.765
